tests/media/video/test_video_import.py

```
@@ -0,0 +1,303 @@
+        # Create test video file
+        test_video_path = self.create_test_video_file("test_input.mp4")
+        self.assertTrue(test_video_path.exists(), "Test video file should be created")
+        
+        # Mock frame cleaning to avoid dependencies
+        with patch.object(VideoImportService, '_ensure_frame_cleaning_available') as mock_frame_cleaning:
+            mock_frame_cleaning.return_value = (False, None, None)
+            
+            # Mock video creation methods with proper center/processor handling
+            with patch('endoreg_db.models.VideoFile.create_from_file_initialized') as mock_create_video:
+                # Create side effect that validates center parameter
+                def create_video_side_effect(file_path, center, processor=None, **kwargs):
```

Ruff (0.14.0)

Unused function argument: `file_path`

(ARG001)